### PR TITLE
{bio}[GCCcore/11.3.0] Health-GPS v1.1.3.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)
requires 
- [x] PR #16211 
- [x] PR #16209 
- [x] PR #16208 
- [x] PR #16207

### CODE REVIEW

**Comment by sassy-crick** [comment]
Maintainer of the program confirms the build is working correctly.

File: easybuild/easyconfigs/h/healthgps/healthgps-1.1.3.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMakeCp'
+
+name = 'healthgps'
```
Comment (boegel) [inline easybuild/easyconfigs/h/healthgps/healthgps-1.1.3.0-GCCcore-11.3.0.eb:6]: `name` should be `Health-GPS` ?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/h/healthgps/healthgps-1.1.3.0-GCCcore-11.3.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/h/healthgps/healthgps-1.1.3.0-GCCcore-11.3.0.eb:6]: Indeed, that is what the maintainer suggested as well. I have updated this and the previous version (PR #15245) to the new name.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7220d4a6a7665ece2d3eec989a28cc63 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**